packages/extension-ui/src/components/RemoveAuth.tsx

```diff
@@ -20,6 +20,7 @@ interface Props {
 const StyledFaviconBox = styled(FaviconBox)`
   box-sizing: border-box;
   margin: 0px auto;
+  max-width: calc(100% - 16px);
```

Comment:
Without this, the outter box would extend to the edges of the window.

Reply:
I think `max-width: calc(100% - 24px);` would look better. On the address below, apart from same trick, there is also 4px inline padding:

![image](https://github.com/Cardinal-Cryptography/aleph-zero-signer/assets/35614528/8b5dc50d-f17b-4367-a181-9b482b4a5496)